x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/components/analytics_tables/analytics_table.scss

```
@@ -0,0 +1,5 @@
+.entSearch__table--noBackground {
```

BEM naming nit - we shouldn't be using `__` or `--` modifiers without a base block/component. Examples: https://css-tricks.com/bem-101/

I'd also personally note that the `entSearch` prefix isn't really required. CSS is loaded in dynamically per-plugin so it's not like we're creating conflicts for other plugins (unless they're manually requiring our SCSS files, in which case it's their own fault :). With that in mind I would suggest a more generic class name.

I would also strongly suggest a comment noting that this CSS can be removed eventually with EUI changes, to help other devs have context and to remind ourselves to remove the code at some point.

Summary of suggestions:

```suggestion
// This CSS can be removed once EUI supports `subdued` panels
.noEuiTableBackground {
```

---

Cool cool. Sounds like the prefixing is a bit of a relic practice. I'll remove it.

---

Update: I ended up moving this CSS into the DataPanel component and baking it into the DataPanel's classes:

https://github.com/elastic/kibana/pull/92939/commits/a98160c802d311dfe56a2c06243479e9ff54bfbf#diff-e5a69ef70ca8d0c72aeefb64aa50636f5c2bb850ce3375a3d322f0144304ef57R1

https://github.com/elastic/kibana/pull/92939/commits/a98160c802d311dfe56a2c06243479e9ff54bfbf#diff-8f297480eb034e309622e67ac786f4ce2b745b202320e350a4c0f26808f5e1fdR43-R45